tests/sentry/sentry_metrics/test_batch.py

```diff
@@ -1954,115 +1954,6 @@ def test_one_org_limited(caplog, settings):
     ]
 
 
-@pytest.mark.django_db
-def test_cardinality_limiter(caplog, settings):
```

Comment:
I'm not sure why this test is dead code? indeed the release health indexer does not have cardinality limiting, but the generic-metrics one does. and it's the same codebase

Reply:
the setup and teardown of this test is the same as tests adjacent to it.  the only difference is the call to the dead function in the middle of it.  removing the unused function (and its call) breaks the test since that functionality doesn't exist any more

Reply:
ah this is about filter_messages. my bad. and yes I just realized cardinality limiter is in relay now